cmd/minishift/cmd/start.go

```diff
@@ -395,8 +395,32 @@ func addActiveProfileInformation() {
 	}
 }
 
+// shouldPerformImageCaching reads the $MINISHIFT_HOME/config/config.json to check
+// if image caching is enabled/disabled from config command and based on it
+// return the behavior of image caching
+// Else perform image caching by default
+func shouldPerformImageCaching() (bool, error) {
+	defaultBehavior := constants.DefaultImageCaching
+
+	config, err := configCmd.ReadConfig()
+	if err != nil {
+		return false, err
+	}
+	for k := range config {
+		if k == configCmd.ImageCaching.Name {
+			defaultBehavior = viper.GetBool(configCmd.ImageCaching.Name)
+		}
+	}
+
+	return defaultBehavior, nil
+}
```

Comment:
Can we move this function to `cmd/util/util.go`? I don't like expending `start.go` day by day. Logic wise 👍